code/game/objects/items/storage/boxes.dm

```diff
@@ -609,6 +609,42 @@
 	playsound(loc, "rustle", 50, 1, -5)
 	user.visible_message("<span class='notice'>[user] hugs \the [src].</span>","<span class='notice'>You hug \the [src].</span>")
 
+/////clown box & honkbot assembly
+obj/item/storage/box/clown
+	name = "clown box"
+	desc = "A colorful cardboard box for the clown"
+	icon_state = "clownbox"
+	illustration = null
+
+/obj/item/stack/sheet/cardboard/attackby(obj/item/I, mob/user, params)

```

Comment:
move this proc to be with the cardboard definition([here](https://github.com/tgstation/tgstation/blob/master/code/game/objects/items/stacks/sheets/sheet_types.dm))

Reply:
Done.